### PR TITLE
First version

### PR DESCRIPTION
**Pull request by NadzeyaZhukovich**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    height: 30px;
+    font-size: 18px;
+    text-align: center;
+    background: lightgray;
+    border-radius: 10px;
+    border: none;
+    margin-right: 10px;
+}
+
+#contactList{
```
Comment (ubcent) [inline style.css]: // sass (scss), less, stylus
.contactCard {
  width: 300px;
}
#contactList {
   .contactCard {
     width: 500px;
   }
}

div.box {
}

div#box {
}

div.box span.item {
}

File: style.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    text-transform: uppercase;
+    font-weight: bold;
+}
+
+#addContact:hover{
+    background: gainsboro;
+    color: #fff;
+}
+
+.btn:{
```
Comment (ubcent) [inline style.css]:
```suggestion
.btn {
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    padding: 0;
+    font-family: Roboto, Times New Roman, serif;
+    color: rgb(71, 71, 71);
+}
+
+header{
+    display: flex;
+    justify-content: center;
+    width: 1100px;
+    margin: 0 auto;
```
Comment (ubcent) [inline style.css]: You shouldn't use margin twice

File: style.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    margin-top: 15px;
+}
+
+#search{
```
Comment (ubcent) [inline style.css]: Please use class instead

File: style.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    height: 20px;
+    border: none;
+    border-radius: 5px;
+    background-color: rgb(233, 232, 232);
+    margin: 2px 0;
+    padding-left: 5px; 
+    font-size: 14px;
+    text-transform: capitalize;
+
+}
```
Comment (ubcent) [inline style.css]: We need an empty line

File: app.js [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    function readContactData() {
+        return {
+            firstName: $firstNameInput.value,
+            lastName: $lastNameInput.value,
+            homePhone: $homePhoneInput.value,
+            mobilePhone: $mobilePhoneInput.value,
+            email: $emailInput.value,
+            photo: $photoInput.value
```
Comment (ubcent) [inline app.js]:
```suggestion
            photo: $photoInput.value,
```

File: app.js [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+        })
+        .then((response) => response.json())
+        .then(createCard);
+    }
+
+    
+    function deleteContact(id, event) {
+        fetch(CONTACTS_URL + '/' + id, {method: 'DELETE'})
+            .then(() => {
+                let $card = event.target.parentElement.parentElement.parentElement;
```
[the remark this answers is not in the document]
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: app.js [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+        $mobilePhoneInput.value = contact.mobilePhone;
+        $emailInput.value = contact.email;
+        $photoInput.value = contact.photo;
+    }
+    //-------------------
+
+    // UI functions
+    function createCard(contact) {
+        let template = document.getElementById('contactCardTemplate').cloneNode(true);
+        template.querySelector('.photo').src = contact.photo;
```
[the remark this answers is not in the document]
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: app.js [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+        $homePhoneInput.value = contact.homePhone;
+        $mobilePhoneInput.value = contact.mobilePhone;
+        $emailInput.value = contact.email;
+        $photoInput.value = contact.photo;
+    }
+    //-------------------
+
+    // UI functions
+    function createCard(contact) {
+        let template = document.getElementById('contactCardTemplate').cloneNode(true);
```
[the remark this answers is not in the document]
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: app.js [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+            updateContact(currentId, contact);
+        } else {
+            addContact(contact);
+        }
+    });
+    
+    $search.addEventListener('keyup', (event) => {
+        let query = event.target.value;
+        if (query.length >= 2 || query.length == 0) {
+            while ($contactList.hasChildNodes()) {
```
[the remark this answers is not in the document]
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: style.css (hunk 2 above)
Comment (NadzeyaZhukovich) [reply style.css]: Fixed

File: style.css (hunk 3 above)
Comment (NadzeyaZhukovich) [reply style.css]: Fixed

File: style.css (hunk 4 above)
Comment (NadzeyaZhukovich) [reply style.css]: Fixed

File: style.css (hunk 5 above)
Comment (NadzeyaZhukovich) [reply style.css]: Fixed

File: app.js (hunk 6 above)
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

**Pull request merged**